### DISCUSSION TITLE
Longhorn - Configuring Default Disk - v1.24.4+rke2r1 - RKE2

### DISCUSSION THREAD

**Discussion by zohebs341** [longhorn/longhorn#4575] [Q&A]
Hi All,

I've deployed a cluster from Rancher UI and want to use longhorn.

I am looking to deploy Longhorn with a custom mount point rather than using the default path. As I've separate mount points available for persistence(PVC).

As I've multiple worker nodes in RKE Cluster, how to configure the defaultDataPath:

As for each VM, the mount point name(Filesystem Type: EXT4) will be different as shown below.

/dev/sda        6.8G   24K  6.5G   1% /mnt/volume_blr1_01
/dev/sda         11G   24K   11G   1% /mnt/volume_blr1_02
/dev/sda         11G   24K   11G   1% /mnt/volume_blr1_03

<img width="794" alt="image" src="https://user-images.githubusercontent.com/62261716/190628113-584b6d1b-ec0c-4cf4-ab1e-52d1eeb63fa5.png">

**Comment by derekbit** [comment]
Please refer to https://github.com/longhorn/longhorn/discussions/4392

**Reply by zohebs341** [reply]
@derekbit Thank you. I've gone through the doc and it served the purpose. I just disabled the default disk and enabled the custom disk for persistence.
